battle-service/src/main/java/org/astashonok/battleservice/constants/ErrorMessage.java

```diff
@@ -0,0 +1,18 @@
+package org.astashonok.battleservice.constants;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class ErrorMessage {
```

Comment:
you can think about error messages hierarchy. it's not very good practice to keep class as placeholder of static variables.


just a brief example (just from top of my head)
`String.format(ErrorMessage.INCORRECT_WINNING_NUMBER_ROW, winningNumberInRow, boardWidth, boardHeight),`
could be replace to 
```
public class IncorrectWinningNumberRowException extends BattleErrorMessage{
private static final String ERROR_PATTERN;
private message;

public IncorrectWinningNumberRowException(winningNumberInRow, boardWidth, boardHeight){
message=String.format(ERROR_PATTERN, winningNumberInRow, boardWidth, boardHeight),
}

String getMessage(){
return message
}
}

new IncorrectWinningNumberRowException(winningNumberInRow, boardWidth, boardHeigh);


```


this will allow to control behaviour more flexible in error message handler advice. (for example to control http status codes)